src/sftp.c

```diff
@@ -1301,8 +1301,7 @@ static ssize_t sftp_read(LIBSSH2_SFTP_HANDLE * handle, char *buffer,
                    allows, expand it! */
 
                 rc = _libssh2_channel_receive_window_adjust(sftp->channel,
-                                                            max_read_ahead*8,
-                                                            1, NULL);
+                                                            max_read_ahead, 1, NULL);
```

Comment:
@bagder On the face of it, this change makes sense.  What was the original rationale for `*8`?